website/src/components/quickstartGuideList/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Head from '@docusaurus/Head';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import Layout from '@theme/Layout';
+import Hero from '@site/src/components/hero';
+import QuickstartGuideCard from '../quickstartGuideCard'
+import styles from './styles.module.css';
+
+const quickstartTitle = 'Quickstart guides'
+const quickstartDescription = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.'
```

Comment:
Hey @runleonarun, if we wanted to use a link in the hero component subheading we would need to add the `dangerouslySetInnerHTML` prop to the subheading which we should avoid using if possible to avoid [cross-site scripting (XSS)](https://en.wikipedia.org/wiki/Cross-site_scripting) attacks. We can add it if we really wanted to but wanted to call that out.

I just pushed the content suggested above without the link, let me know what you think.